src/VIPServer.java

```diff
@@ -107,24 +107,6 @@ public void main(String[] args) throws HostFailureException, HostNotFoundExcepti
 					// SHOULD BE REPLACED BY
 					// job.carryOn();
 				} else {
-					if (!timer) {
```

Comment:
Counting the number of particles in the static mode is not a problem (we actually do it in the real workflow for information purposes). 

I agree with the changes required to distinguish the 2. 

Regarding the last question, since in simulation we actually inject execution times, the number of particles is not really important. So I don't think we need to bother with getting the total number of particles and dividing it among workers. The way it is currently implemented seems fine to me.